week-2-practise/level-2/2-validation.js

```diff
@@ -4,7 +4,9 @@
 // - is less than or equal to 100
 // Tip: write other small functions for each requirement
 
-function validate(num) {}
+function validate(num) {
+  return typeof num === "number" && num > 0 && num <= 100;
+}
 
```

Comment:
i found that if we use the Math.sign() function to returns the sign of a number it will return 0 for o  because of this i didnt  include it in the function but if i will concide it positive than i will add num>=0
```suggestion

```
```suggestion
console.log(Math.sign(0));
// expected output: 0
```

Reply:
```suggestion
function validate(num) {
  return typeof num === "number" && num >= 0 && num <= 100;
}
```